qiskit/transpiler/passes/optimization/commutative_inverse_cancellation.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2017, 2021.
+# (C) Copyright IBM 2023, 2024.
```

Comment:
I've seen different opinions on copyrights, but I think all of them agreed that, if two dates are given, the first year must be Qiskit's creation year 🙂 Some say it should be `<creation of Qiskit> <creation of file>`, others `<creation of Qiskit> <last year of change>`, not sure which is truly correct.

Reply:
I thought it's either a single year ``<creation of file>``, or a range of years ``<creation of file>, <last modification of file>``. But I am not longer sure about this, so changed it to ``2017, 2024``.